backend/infrahub/core/schema/basenode_schema.py

```diff
@@ -62,6 +63,17 @@ def __hash__(self) -> int:
         Be careful hash generated from hash() have a salt by default and they will not be the same across run"""
         return hash(self.get_hash())
 
+    def to_dict(self) -> dict:
```

Comment:
I went down a bit of a rabbit hole on this one regarding the best way to serialize `Enum` into string when doing a `model_dump()`
I looked at `model_serializer` but it created a lot of additional issue in test, I think there are other part of the code that expect Enum to be Enum in a dump ... 
For now it's working as it but I have the feeling that a larger cleanup would be good around Enum